scripts/run-system-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
I think this script will be changed in the future to allow:
1. Single test case execution
2. Run tests without coverage check

so anything we can do now without big effort to make implementation of those features easier in the future, let's consider it.

Reply:
I have single test execution on my TODO list, so I added optional argument for single test execution - hope it will be enough